website/docs/sat/building-sat.md

```diff
@@ -0,0 +1,32 @@
+# Building Sat
+If you'd like to build Sat yourself, clone this repo and run:
+```bash
+# On M1, you may need to run `export DOCKER_BUILDKIT=0`
+make docker run
+```
+This will build the `suborbital/sat:dev` Docker image and then launch it using the same `hello-echo` module.
+
+## Stdin mode
```

Comment:
I think the stdin mode and run from url instructions should go on the `using sat` page